gui-wm/labwc/labwc-0.7.2.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 2023 Aisha Tammy and B. Gazotti
+# Copyright 2021 Aisha Tammy
```

Comment:
Please re-add their name, and if you wish please add your's :)

Reply:
My bad on missing that! Not sure why but git seems to think that I was renaming 0.7.0 to 0.7.2 because I deleted 0.7.0. I'll re-add this now.